store/migration/dev/LATEST.sql

```diff
@@ -926,13 +926,13 @@ CREATE TABLE repository (
     base_directory TEXT NOT NULL DEFAULT '',
     -- The file path template for matching the committed migration script.
     file_path_template TEXT NOT NULL DEFAULT '',
+    -- If enable the SQL review CI in VCS repository.
+    enable_sql_review_ci BOOLEAN NOT NULL DEFAULT false,
```

Comment:
Why just moving around column

Reply:
This is to be consistent with prod/LATEST.sql. Since we cannot change prod, let's change dev.